### PR TITLE
Generate 2022 04 01 desktopvirtualization

### PR DESCRIPTION
**Pull request by ryanosh**
## Description

generated powershell modules for desktopvirtualization api version 2022-04-01-preview and updated modules and tests for scalingplan and scalingplanpooledschedule accordingly

## Checklist

- [ X] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from the scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** change version of module in pull request

### CODE REVIEW

File: src/DesktopVirtualization/docs/New-AzWvdHostPool.md [hunk 1]
```diff
@@ -103,6 +106,69 @@ This command creates a Windows Virtual Desktop HostPool in a Resource Group.
 
 ## PARAMETERS
 
+### -AgentUpdateMaintenanceWindow
```
Comment (isra-fel) [inline src/DesktopVirtualization/docs/New-AzWvdHostPool.md:109]: These new parameters could really use an example

File: src/DesktopVirtualization/examples/New-AzWvdScalingPlan.md [hunk 2]
```diff
@@ -1,33 +1,45 @@
 ### Example 1: Create a Windows Virtual Desktop Scaling Plan
 ```powershell
-New-AzWvdScalingPlan `
+PS C:\> New-AzWvdScalingPlan `
```
Comment (isra-fel) [inline src/DesktopVirtualization/examples/New-AzWvdScalingPlan.md:3]: Please remove the command prompt as we wish the example to be one-click runable
```suggestion
New-AzWvdScalingPlan `
```

**Review by isra-fel — CHANGES_REQUESTED** [review]
- Please remove `src/DesktopVirtualization/test/Az.DesktopVirtualization-TestResults.xml`. This file should not be checked in.
- Please see my other inline comments. Note that some of them may apply to multiple files.

**Comment by isra-fel** [comment]
Please also resolve the conflicts to unblock CI.

**Comment by isra-fel** [comment]
Kindly ping @ryanosh

File: src/DesktopVirtualization/docs/New-AzWvdHostPool.md (hunk 1 above)
Comment (ryanosh) [reply src/DesktopVirtualization/docs/New-AzWvdHostPool.md:109]: I  updated the example to include the new agent update parameters

**Comment by wyunchi-ms** [comment]
Please resolve the conflicts. @ryanosh

**Comment by isra-fel** [comment]
Closing the PR for lack of activity. Feel free to reopen to continue working on it. Thanks 😀

**Pull request closed without merging**